### PR TITLE
panther: display extra installation steps and enable the screenshot on failure feature

### PR DESCRIPTION
**Pull request by dunglas**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | n/a

Enables https://github.com/symfony/panther/pull/392.

### CODE REVIEW

File: symfony/phpunit-bridge/4.1/.env.test [hunk 1]
```diff
@@ -3,3 +3,4 @@ KERNEL_CLASS='App\Kernel'
 APP_SECRET='$ecretf0rt3st'
 SYMFONY_DEPRECATIONS_HELPER=999999
 PANTHER_APP_ENV=panther
+PANTHER_ERROR_SCREENSHOT_DIR=./tests/error-screenshots
```
Comment (weaverryan) [inline symfony/phpunit-bridge/4.1/.env.test:6]: What about `./var/error-screenshots`? It's already ignored

Comment (dunglas) [reply symfony/phpunit-bridge/4.1/.env.test:6]: Isn't it harder to discover? I was hesitating with simply `./error-screenshots`, at the root.

Comment (nicolas-grekas) [reply symfony/phpunit-bridge/4.1/.env.test]: I like `var/something` also. Doc will tell where this is anyway.

Comment (dunglas) [reply symfony/phpunit-bridge/4.1/.env.test]: @nicolas-grekas yes, I updated the PR accordingly. It looks like my comment saying that I changed my mind has been lost :/ I have issues with GitHub these days.

**Review by chalasr — APPROVED** [review]

**Review by nicolas-grekas — APPROVED** [review]

**Pull request merged**